src/providers/analytics/analytics.model.ts

```diff
@@ -42,6 +42,8 @@ export enum AnalyticsEventCategories {
   POST_TEST = 'post-test',
   TEST_REPORT = 'test report',
   TERMINATION = 'test termination',
+  PRACTICE_TEST = 'practice test',
```

Comment:
If I was being picky practice test is not a screen name ;)

Reply:
As you now know, it's actually under `AnalyticsEventCategories` ;)